test/passing/tests/break_separators-after_docked.ml.ref

```diff
@@ -322,8 +322,7 @@ let foooooooooooo =
 let foooooooooooo =
   {
     foooooooooooooo with
-    (* foooooooooooooooo fooooooooooooooooooooooooo
-       foooooooooooooooooooooo *)
+    (* foooooooooooooooo fooooooooooooooooooooooooo foooooooooooooooooooooo *)
```

Comment:
This is an unwrapped comment. It's not obvious how the break appeared with the previous formatting code.

Reply:
Why is it unwrapped? It is neither a banner comment nor is opened with `(*=`, is there some other mechanism to make it unwrapped?

Reply:
Right! I forgot that the tests were run with `wrap-comments` enabled. The new formatting is fine then.

Reply:
You mean this PR changes the settings used to run the tests? That makes it hard to tell if there are regressions. We also do want to test the comment wrapping functionality, probably more thoroughly than the non-wrapping functionality, since it is nontrivial.

Reply:
The tests are run with `wrap-comments` since before this PR. This is something that I frequently forget about. This diff shows a fix and not a regression, my first comment was wrong.

> We also do want to test the comment wrapping functionality, probably more thoroughly than the non-wrapping functionality, since it is nontrivial.

That's what we always did but I don't agree with the reason. The wrapping algorithm is trivial.
Perhaps you are talking about detecting which comments should be wrapped/preserved/asterisk prefixed ? This is now unrelated to the `wrap-comments` option.

Reply:
So no regression if I understand correctly?

Reply:
Yes, this is fine.